src/scripts/crank.ts

```diff
@@ -34,24 +36,25 @@ const {
   MAX_UNIQUE_ACCOUNTS,
   CONSUME_EVENTS_LIMIT,
   CLUSTER,
-  HIGH_FEE_MARKETS,      // markets to apply a priority fee for
   PRIORITY_QUEUE_LIMIT,  // queue length at which to apply the priority fee
-  DEFAULT_CU_PRICE,     // extra microlamports per cu for any market
   PRIORITY_CU_PRICE,     // extra microlamports per cu for high fee markets
   PRIORITY_CU_LIMIT,     // compute limit
   POLL_MARKETS, // optional for using Top markets
+  MAX_TX_INSTRUCTIONS,   // max instructions per transaction
+  MIN_FEE_BUMP,          // extra microlamports per cu for any transaction
+  BUMP_MARKETS,          // input to add comma seperated list of markets that force fee bump
 } = process.env;
 
 const cluster = CLUSTER || 'mainnet';
 const interval = INTERVAL || 1000;
 const maxUniqueAccounts = parseInt(MAX_UNIQUE_ACCOUNTS || '10');
 const consumeEventsLimit = new BN(CONSUME_EVENTS_LIMIT || '30');
-// FIXME: make this a list of pubkeys so that it works with POLL_MARKETS option
-const priorityMarkets: number[] = JSON.parse(HIGH_FEE_MARKETS || "[0,1]");
+const bumpMarkets = BUMP_MARKETS ? BUMP_MARKETS.split(',') : [] ;
 const priorityQueueLimit = parseInt(PRIORITY_QUEUE_LIMIT || "100");
-const defaultPriorityCuPrice = parseInt(DEFAULT_CU_PRICE || "0");
+const minFeeBump = parseInt(MIN_FEE_BUMP || "0");
 const priorityCuPrice = parseInt(PRIORITY_CU_PRICE || "100000");
-const CuLimit = parseInt(PRIORITY_CU_LIMIT || "50000");
+const CuLimit = parseInt(PRIORITY_CU_LIMIT || "250000");
```

Comment:
you increased this despite multiplying by number of markets cranked. I think this should stay at 50k

Reply:
https://github.com/SpaceMonkeyForever/openbook-cranker/pull/4/commits/63aeb9ca680dc643371bac6cad64a2c5b3da7662